flixel/util/FlxStringUtil.hx

```diff
@@ -262,14 +262,8 @@ class FlxStringUtil
 	 */
 	public static function getDomain(url:String):String
```

Comment:
Fair point.  I always worry that I end up overthinking and over-engineering things.  And often it is the case of low-hanging fruit where I might think, "It would only take a few minutes to do it right, so better to get it done now and have it done right."  As things appear to be stable enough for the time-being, we can always revisit this later as needed.

Many thanks for your feedback.  `(:`